server/datastore/mysql/migrations/tables/20230719152138_ScheduledQueriesDataMigrator.go

```diff
@@ -0,0 +1,268 @@
+package tables
+
+import (
+	"database/sql"
+	"fmt"
+	"strconv"
+	"strings"
+
+	"github.com/fleetdm/fleet/v4/server/fleet"
+)
+
+func init() {
+	MigrationClient.AddMigration(Up_20230719152138, Down_20230719152138)
+}
+
+func Up_20230719152138(tx *sql.Tx) error {
```

Comment:
I believe we also want to remove all query stats: `DELETE FROM scheduled_query_stats;` and `DELETE FROM aggregated_stats WHERE type = 'query' OR type = 'scheduled_query';`.

/cc @zhumo